packages/backend/src/postgres.ts

```diff
@@ -262,6 +262,33 @@ export const entities = [
 
 const log = process.env.NODE_ENV !== 'production';
 
+/**
+ * Execute query with extended timeout.
+ * This can be used to execute long-running queries like deleting many rows CASCADE-ly.
+ *
+ * TODO: consider remove this function when we removed CASCADE delete.
+ */
+export async function extendTimeoutQuery(dataSource: DataSource, query: (manager: EntityManager) => Promise<void>): Promise<void> {
+	const queryRunner = dataSource.createQueryRunner('master');
+	const manager = dataSource.createEntityManager(queryRunner);
+	const extendedTimeout = 1000 * 100; // 100 sec for now. How long should it be?
```

Comment:
```suggestion
	const extendedTimeout = 1000 * 100; // 100 sec for now. How long should it be?
```

もし試す方でタイムアウトを無効化する場合はこちらを0にしてください　